methods/method.go

```diff
@@ -1,10 +1,25 @@
 package methods
 
-import "github.com/prometheus/prometheus/pkg/rulefmt"
+import (
+	"time"
+
+	"github.com/prometheus/common/model"
+	"github.com/prometheus/prometheus/pkg/rulefmt"
+)
+
+type Window struct {
+	Duration     model.Duration `yaml:"duration"`
+	Consumption  float64        `yaml:"consumption"`
+	Notification string         `yaml:"notification"`
```

Comment:
@pedrokiefer could you create a `type NotificationType string` with two instances `NotificationTicket` and `NotificationPage`?

Reply:
done @pedrokiefer